src/Components/Pages/TransferPage.tsx

```diff
@@ -48,9 +51,55 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
   createStyles({
     root: {
       fontFamily: styles.primaryFont,
-      padding: constants.generalUnit * 3,
+      padding: constants.generalUnit * 2.5,
       position: "relative",
       backgroundColor: "white",
+      minWidth: 460,
+    },
+    wrapper: {
+      position: "relative",
+      background: "white",
+    },
+    header: {
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      marginTop: constants.generalUnit * 2,
+    },
+    logo: {
+      width: 26,
+      height: 26,
+    },
+    headerText: {
+      fontWeight: "bold",
+      fontSize: 30,
+      lineHeight: `${constants.generalUnit * 3}px`,
+      color: "black",
+      paddingLeft: 12,
+    },
+    selectArea: {
+      padding: `${constants.generalUnit * 2.5}px ${
```

Comment:
remove `px`

Reply:
Need to mention the measuring units for shorthand properties.